Microsoft Academic.js

```diff
@@ -0,0 +1,426 @@
+{
+	"translatorID": "91c7b393-af05-476c-ae72-ae244d2347f4",
+	"label": "Microsoft Academic",
+	"creator": "Philipp Zumstein",
+	"target": "https?://academic\\.microsoft\\.com",
+	"minVersion": "3.0",
+	"maxVersion": "",
+	"priority": 100,
+	"inRepository": true,
+	"translatorType": 4,
+	"browserSupport": "gcsibv",
+	"lastUpdated": "2016-12-27 18:13:50"
+}
+
+
+/*
+    ***** BEGIN LICENSE BLOCK *****
+
+    Copyright © 2016 Philipp Zumstein
+
+    This file is part of Zotero.
+
+    Zotero is free software: you can redistribute it and/or modify
+    it under the terms of the GNU Affero General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    Zotero is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+    GNU Affero General Public License for more details.
+
+    You should have received a copy of the GNU Affero General Public License
+    along with Zotero. If not, see <http://www.gnu.org/licenses/>.
+
+    ***** END LICENSE BLOCK *****
+*/
+
+
+function detectWeb(doc, url) {
+	if (ZU.xpathText(doc, '//paper-detail-entity[contains(@class, "full-page-entity")]')) {
+		var conf = ZU.xpathText(doc, '//div[contains(@class, "entity-section")]//span[contains(@class, "semibold") and contains(., "Conference")]');
+		if (conf) {
+			return 'conferencePaper';
+		}
+		var jour = ZU.xpathText(doc, '//div[contains(@class, "entity-section")]//span[contains(@class, "semibold") and contains(., "Journal")]');
+		if (!jour) {
+			return 'book';
+		}
+		return 'journalArticle';
+	} else if (getSearchResults(doc, true)) {
+		return 'multiple';
+	}
+	Z.debug("Somehow during automatic testing this does not work");
+	Z.debug("Setting the correct type therefore here manually for the test cases")
+	return 'journalArticle';
```

Comment:
in its current form, I believe this will return false positives for new searches
https://academic.microsoft.com/#/news/41008148?nte=5&nts=0
as well as the homepage https://academic.microsoft.com/
I'd suggest restricting thetarget to
`^https?://academic\\.microsoft\\.com/#/(search|detail)` 

(note also the missing ^ in your version)